recipes/pal2nal/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "pal2nal" %}
-{% set version = "14" %}
-{% set sha256 = "0e458d38571ab6d61078047a9e80d843bb2e733414eeff8b1144d1c6bc13848f" %}
+{% set version = "14.1" %}
+{% set sha256 = "94c77f793d7b8d2b280eaf6245b50ce2cd38f35cb12a8e7cd3d0d9dd58284ce4" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: http://www.bork.embl.de/pal2nal/distribution/pal2nal.v14.tar.gz
+  url: https://github.com/liaochenlanruo/PAL2NAL/archive/v{{ version }}.tar.gz
 

```

Comment:
Also, please remove the extra line between `url` and `sha256`, and `host` and `run`, and `test` and `about`.